src/main/java/no/nav/familie/inntektsmelding/server/authz/api/PolicyType.java

```diff
@@ -0,0 +1,7 @@
+package no.nav.familie.inntektsmelding.server.authz.api;
+
+public enum PolicyType {
+    INNTEKTSMELDING,
+    PORTAL,
```

Comment:
PORTAL er et generisk uttrykk - mener vi fagsystem?

Reply:
Tenkte mer interaksjon mot fager - så kanskje bruke en mer generisk policy som SYSTEM siden det eneste den sjekken gjør er en sjekk om det er fagsystemet som kaller og at det er en WRITE operasjon (foreløpig).

Reply:
Alltid vanskelig med navn på disse tingene. Hva = INNTEKTSMELDING for alle formål. Hvem = Arbeidsgiver eller Saksbehandler